index.bs

```diff
@@ -132,29 +132,29 @@ header in a response. This sets the origin's [=/client hints set=] to an empty s
 
 Note: As the cache can only be modified by the top-level frame, it is considered to be partitioned.
 
-To <dfn>add a new Accept-CH cache entry</dfn> to the [=Accept-CH cache=],
-given an [=accept-ch-cache/origin=] |origin| and a [=/client hints set=] |hintSet|,
-[=map/set=] [=Accept-CH cache=][|origin|] to |hintSet|.
-
-When asked to <dfn>retrieve the client hints set</dfn> given a |settingsObject|:
+When asked to <dfn>update the settings object's Client Hints set from the cache</dfn> given a |settingsObject|:
```

Comment:
sorry for the bikeshed... we could make this shorter, something like `<dfn>update the client hints set from cache<dfn> with |settingsObject|`

(and I think we want lowercase client hints set, since that's how we spell the thing https://wicg.github.io/client-hints-infrastructure/#concept-client-hints-set)